docs/reference/query-languages/esql/_snippets/commands/layout/change_point.md

```diff
@@ -4,12 +4,9 @@
 The `CHANGE_POINT` command requires a [platinum license](https://www.elastic.co/subscriptions).
 :::
 
-::::{warning}
-This functionality is in technical preview and may be
-changed or removed in a future release. Elastic will work to fix any
-issues, but features in technical preview are not subject to the support
-SLA of official GA features.
-::::
+```yaml {applies_to}
+stack: preview 9.1.0
```

Comment:
```suggestion
stack: preview 9.0.0
```

CHANGE_POINT is preview since 9.0.0 right?

Reply:
Going by this [PR](https://github.com/elastic/elasticsearch/pull/126407) we declared it tech preview in 9.1.

Cc @jan-elastic

Reply:
Nvm, added in https://github.com/elastic/elasticsearch/pull/120998 which was merged in February and didn't make it into 9.0 as far as I can tell.

Reply:
Ooh, right, very important catch, thanks!